client/my-sites/people/followers-list/followers.jsx

```diff
@@ -96,16 +90,13 @@ class Followers extends Component {
 	};
 
 	noFollowerSearchResults() {
-		return (
-			this.props.fetchInitialized &&
-			this.props.query.search &&
-			! this.props.followers.length &&
-			! this.props.fetching
-		);
+		const { search, followers, isFetching } = this.props;
+		return search && ! followers.length && ! isFetching;
```

Comment:
Could we reuse `this.siteHasNoFollowers()` here?

```suggestion
		return search && this.siteHasNoFollowers();
```

Reply:
done via https://github.com/Automattic/wp-calypso/pull/50378/commits/959dab9b048121f9564d0c809eecffee6fd4255b – I had to remove unused properties one line above which is why I didn't apply that suggestion.